src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -141,7 +145,7 @@ class SwitchHelper {
     }
 
     static List<Long> getDefaultCookies(Switch sw) {
-        def swProps = northbound.get().getSwitchProperties(sw.dpId)
+        def swProps = getSwitchProperties(sw.dpId)
```

Comment:
This change should break many tests that rely on `getDefaultCookies` method to verify switch state after some switch property was changed